### ISSUE TITLE
Noti without payload

### ISSUE THREAD

**Issue by btro1404** [Tom-Hirschberger/MMM-MQTTbridge#22]
I see some notifications without payload. Because I want to publish them by mqtt I say `payloadValue: ''` in `notiDictionary.js`. The result is: MQTTbridge publish a mqtt -topic without any payload.

Do you see a possibility to add a payload? Maybe in a setup similar to noempty `payloadValue`:
```
var notiHook = [
  {
    notiId: "SCREEN_ON",
    notiPayload: [
      {
        payloadValue: '',
        notiMqttCmd: ["SCR_1"]
      },
    ],
  },
];
var notiMqttCommands = [
    {
        commandId: "SCR_1",
        mqttTopic: "stat/magicmirror/screen",
        mqttMsgPayload: '{"state":"OFF"}'
    },
];

```
?

Regards, Bernd

**Comment by Tom-Hirschberger** [comment]
Hi,

i just took over the ownership of the module.
Is this problem still present?
Can you give me some more information what exactly is the problem?

**Comment by Tom-Hirschberger** [comment]
I think i realized the problem today. 
You want to react to a notification which has no payload. But if you configure payloadValue: '' the mqttMsgPayload configured in the command will be ignored.

I am working on a new version of the module currently which provides QoS and Retain settings and fixes this problem.

**Comment by btro1404** [comment]
Hi Tom,

`You want to react to a notification which has no payload. But if you configure payloadValue: '' the mqttMsgPayload configured in the command will be ignored.`

Right - that's it!

Gruß Bernd

**Comment by Tom-Hirschberger** [comment]
Released version 2.0.0 a few seconds ago. Your problem should be fixed with this release.
Be sure to save your dict files before update and read the sections about the dicts in the updated readme.
